### PR TITLE
Add builder dev report for 2017-07-17

### PR DESCRIPTION
**Pull request by tonistiigi**
@vieux @AkihiroSuda @thaJeztah 

Signed-off-by: Tonis Tiigi <tonistiigi@gmail.com>

### CODE REVIEW

File: reports/builder/2017-07-17.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Following features were added last week:
+
+#### Git source
+
+Source code from git repositories can now be accessed directly, similarly for images can be accessed, without the need to execute `git clone`. This has many performance and caching advantages. It accesses the remote repository using shallow fetches to only pull the required data and a uses a shared bare repository for intermediate cache between build invocations. The instruction cache for the git source is based on a commit hash and not string arguments. This means that you can always be sure that you are building the correct source and that you never build the same source twice.
+
+#### Containerd exporter
+
+Exporters are used for getting build artifacts out of buildkit. The first exporter that was implemented allows exposing the image to containerd so it can be run with `ctr` or pushed with `dist` tool. `buildctl` has `--exporter` flag for specifying the exporter and `--exporter-opt` for custom values passed to the exporter. In the case of image exporter an image name can be specified.
```
Comment (AkihiroSuda) [inline reports/builder/2017-07-17.md]: No dist tool anymore

Comment (tonistiigi) [inline reports/builder/2017-07-17.md]: There is in `alpha0` that is supposed to be the "supported" version. Anyway, I checked that everything works with master containerd as well and updated the examples.

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
@vieux @thaJeztah PTAL?

**Review by thaJeztah — APPROVED** [review]
LGTM! sorry for the delay

**Pull request merged**